microsoft/runbook/qemu/CBL-Mariner.yml

```diff
@@ -0,0 +1,6 @@
+name: qemu default
+include:
+  - path: ./qemu.yml
+variable:
+  - name: extra_user_data
+    value: "$(runbook_dir)/CBL-Mariner/user-data"
```

Comment:
It can be a relative path and resolve it in platform, so it doesn't need the runbook_dir variable.

Reply:
It is worth noting that `runbook_dir` is for the current runbook, not the base runbook. This has the nice advantage of making it easier to organize runbooks in directories. The orchestrator code can't mimic this behavior because all the data has been merged and the context of which runbook a value came from has been lost.

Though I guess I don't actually need this extra complexity at the moment. So, I am happy to remove it.